packages/react/src/components/IconSwitch/_icon-switch.scss

```diff
@@ -6,72 +6,95 @@ $icon-switch-size-large: $spacing-09;
 
 .#{$iot-prefix}--icon-switch {
   border: 0;
+  border-top: 1px solid $ui-05;
+  border-bottom: 1px solid $ui-05;
+  position: relative;
+
+  &:focus {
+    box-shadow: inset 0 0 0 2px $focus, inset 0 0 0 3px $ui-01;
+
+    &,
+    & + .#{$iot-prefix}--icon-switch {
+      .#{$iot-prefix}--icon-switch__divider:after {
+        display: none;
+      }
+    }
+  }
+
+  &:hover {
+    // hide the divider on hover
+    &:not(.#{$iot-prefix}--icon-switch--light) {
+      .#{$iot-prefix}--icon-switch__divider:after {
+        display: none;
+      }
+
+      // hide the divider beside it on hover to prevent "fuzziness" of the hover state
```

Comment:
hehe

Reply:
:-) There are so many random little rules in there, I figured it good to label some of them.